### DISCUSSION TITLE
Bundled LAME MP3 Encoder vs LAME MP3 Command Line

### DISCUSSION THREAD

**Discussion by Blushock** [enzo1982/freac#485] [Help & Support] [answered]
https://freac.org/manual/en/howto.html#anchor_enc -> I noticed that one of the options is the LAME Command Line Encoder, but fre:ac already has the lame codec by default. Why is there a second option? Thanks in advance.

**Comment by enzo1982** [comment, marked as answer]
It's there so 3rd party builds of LAME can easily be used with fre:ac. Such as builds from [RareWares](https://www.rarewares.org/mp3-lame-bundle.php) or the [tuned builds from HydrogenAudio user halb27](https://hydrogenaud.io/index.php/topic,112036.0.html). So mainly for users who want to experiment with different versions of LAME.
